build.gradle

```diff
@@ -96,6 +96,12 @@ allprojects { subproj ->
 		mavenCentral()
 		maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
 	}
+	buildscript {
+		repositories {
+			// mavenLocal()
+			mavenCentral()
+		}
+	}
```

Comment:
:-1: 
Sorry, there was a change in _spotless_ to lookup the external plugins (like Eclipse based formatters) from the repositories in _buildscript_. The repositories are not inherited like methods ore externals. Hence [configuration injection](https://docs.gradle.org/3.3/userguide/organizing_build_logic.html#sec:injected_configuration) is required.